File: src/main/java/com/cfblj/carrental/service/CarAppointService.java
package com.cfblj.carrental.service;

import com.baomidou.mybatisplus.extension.service.IService;
import com.cfblj.carrental.model.CarAppoint;
import com.cfblj.carrental.utils.Pages;

public interface CarAppointService extends IService<CarAppoint> {
    Pages getCarAppointPage(CarAppoint carAppoint, int curPage, int size);

    CarAppoint getCarAppointById(String id);

    void addCarAppoint(CarAppoint carAppoint);
}

File: src/main/resources/static/js/carrental/service/carAppointService.js
app.service('carAppointService', function ($http) {

    //查询所有车辆预约信息，带分页
    this.getCarAppointPage=function (curPage, size, searchCarAppoint) {
        return $http.post('/carAppoint/getCarAppointPage?curPage='+curPage+'&size='+size, searchCarAppoint);
    }

    this.getCarAppointById=function (id) {
        return $http.get('/carAppoint/getCarAppointById?id='+id);
    }


});